File: practicas/biblio/horario.py
class Horario:
    def __init__(self, dias_apertura, hora_apertura, hora_cierre):
        self.dias_apertura = dias_apertura
        self.hora_apertura = hora_apertura
        self.hora_cierre = hora_cierre

    def mostrar_horario(self):
        print(f"Horario: {self.dias_apertura} de {self.hora_apertura} a {self.hora_cierre}")

File: practicas/bibli/horario.java
package practicas.bibli;


public class horario {
    private String diasApertura;
    private String horaApertura;
    private String horaCierre;

    public horario(String diasApertura, String horaApertura, String horaCierre) {
        this.diasApertura = diasApertura;
        this.horaApertura = horaApertura;
        this.horaCierre = horaCierre;
    }

    public void mostrarHorario() {
        System.out.println("Horario: " + diasApertura + " de " + horaApertura + " a " + horaCierre);
    }
}